src/client.ts

```
@@ -840,17 +840,36 @@ export class BikeTagClient {
 
     switch (options.source) {
       case AvailableApis.aws:
-        clientMethod = clientMethod.bind({
-          getQueue: this.getPassthroughApiMethod(api.getQueue, client),
-          getTags: this.getPassthroughApiMethod(api.getTags, client),
-          updateTag: this.getPassthroughApiMethod(api.updateTag, client),
-          uploadTagImage: api.uploadTagImage.bind({
+        const getTags = this.getPassthroughApiMethod(api.getTags, client)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)